.envs/.django

```diff
@@ -14,5 +14,5 @@ CELERY_FLOWER_USER=debug
 CELERY_FLOWER_PASSWORD=debug
 
 # Opensearch
-OPENSEARCH_HOST=opensearch:9200
+OPENSEARCH_HOST=127.0.0.1:9200
```

Comment:
Why changing this?

Reply:
because it doesn't work otherwise, as we discussed yesterday just before leaving home. I am trying to understand why  env is not taken from test.py file